openfisca_france/parameters/prelevements_sociaux/contributions_sociales/crds/activite/taux.yaml

```
@@ -8,7 +8,7 @@ metadata:
   last_review:
     1998-01-01
   reference:
-  date_parution_jo:
+  date_of_issue_in_oj:
```

Mettre à jour vers la dernière version du nom utilisé ailleurs dans cette PR ? 😊 
```suggestion
  official_journal_date:
```